File: Web-Ecommerce/Front/shared/types/feedback.ts
export interface Feedback {
  id: number
  userId: number
  type: number
  typeText: string
  title: string
  content: string
  contact: string
  images: string
  status: number
  statusText: string
  adminReply: string | null
  adminId: number | null
  username: string
  userEmail: string
  handleTime: string | null
  createTime: string
  updateTime: string
}

export interface FeedbackSubmitForm {
  type: number
  title: string
  content: string
  contact?: string
  images?: string[]
}

File: Web-Ecommerce/Back/src/main/java/com/ecommerce/entity/Feedback.java
package com.ecommerce.entity;

import com.baomidou.mybatisplus.annotation.*;
import lombok.Data;
import java.time.LocalDateTime;

@Data
@TableName("feedback")
public class Feedback {
    @TableId(type = IdType.AUTO)
    private Long id;
    private Long userId;
    private Integer type;
    private String title;
    private String content;
    private String contact;
    private String images;
    private Integer status;
    private String adminReply;
    private Long adminId;
    private LocalDateTime handleTime;
    @TableField(fill = FieldFill.INSERT)
    private LocalDateTime createTime;
    @TableField(fill = FieldFill.INSERT_UPDATE)
    private LocalDateTime updateTime;

    @TableField(exist = false)
    private String typeText;
    @TableField(exist = false)
    private String statusText;
    @TableField(exist = false)
    private String username;
    @TableField(exist = false)
    private String userEmail;
}
